tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -132,7 +132,7 @@ Get Spawner Environment Variable Value
 
 Spawn Notebook
     [Documentation]  Start the notebook pod spawn and wait ${spawner_timeout} seconds (DEFAULT: 600s)
-    [Arguments]  ${spawner_timeout}=600 seconds  ${same_tab}=${True}
+    [Arguments]  ${spawner_timeout}=600 seconds  ${same_tab}=${True}  ${autoscale}=${False}
```

Comment:
Could you add in [Documentation] what is the ${autoscale} parameter for?  Maybe `${wait_for_autoscaling}` or `${expect_autoscaling}` would be more descriptive